images/docker-compose-mca.yml

```diff
@@ -8,6 +8,6 @@ services:
       labels:
         org.opencontainers.image.source: https://github.com/cnpem/epics-in-docker
       args:
-        REPONAME: mca
+        REPONAME: epics/modules/mca
```

Comment:
To be honest, I don't like very much the idea of reusing `REPONAME`. It seems awkward to me. I'd rather have them specified by a variable whose name makes sense at all.

I did so because it was easier to test other things first. Let me know what you think.

Reply:
That's fair. Do you think it would lead to special casing these modules in the image Dockerfile?

Reply:
> Do you think it would lead to special casing these modules in the image Dockerfile?

They kinda already have such special handling... all of them are no-build targets. They of course currently have almost the same (compose variables) API of other targets. I say "almost" because they diverged by not using REPONAME before, and now by not meaning "base name to where to clone the repository" but rather "location inside /opt where important artifacts live".

I'm not sure if I answered your question at all, but I do feel like we have always being dealing with a special case.

Reply:
I guess I am essentially wondering if we shouldn't use some other name in REPONAME, since these images are a special case.

Reply:
I changed this to `APP_DIRS`. It should encode full paths now. It is named after a plural noun; after all, we can have multiple directories that we want to ensure are properly working. This should work as a palliative measure for spurious cases where we clean up too much. Any directory listed there will have its belonging module removed from pruning (both module-based and directory-based).

Maybe we could be even more explicit about its usage regarding "not pruning it too much and keeping its executable binaries working". Any opinion and suggestion about that?